libs/ngxtension/mapfilter/project.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "ngxtension/mapfilter",
```

Comment:
nit: Not sure about the naming scheme but should the file be `ngxtension/map-filter` instead (hyphen in middle?)

Reply:
I don't know if there is an enforced/preferred naming convention for the files...

Reply:
Ok @nartc I'll rename all files in next commit
PS: Do you have any suggestion about the @ts-ignore problem for CI?

Reply:
You can add `noImplicitReturns: false` to `tsconfig.spec.json`